vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -679,7 +684,7 @@ - (void)onReconcileComplete:(ledger::Result)result viewingId:(const std::string
                        userInfo:info
                  notificationID:[NSString stringWithFormat:@"contribution_%@", viewingId]];
   }
-
+  [self fetchBalance:nil];
```

Comment:
This is still being called outside of the success check above

Reply:
Moved inside success block. I was with the understanding that it should be where other observers are called, as balance report will not be updated it there was an error?

Reply:
yeah I understand, I just thought of putting other observers. But you are right inside success is the better place